ui/src/app/views/registration-dashboard.tsx

```diff
@@ -213,24 +227,71 @@ export class RegistrationDashboard extends React.Component<RegistrationDashboard
     }
   }
 
+  async setAllModulesBypassed(isBypassed: boolean) {
+    this.setState({bypassInProgress: true});
+
+    // TypeScript enum iteration is nonfunctional
```

Comment:
so sad :(

Reply:
Typescript enum iteration normally works, but it appears to have relatively broken behavior when the value types are `any`, and this is how ts fetch codegen generates its enums for some reason. The normal way to write the enum would  be to use a string value.

Reply:
Thanks – I can try that out as a follow-up (I'd rather not touch the AccessModule enum as part of this PR).